internal/server/v3/nodesearch/golden/node_search_test.go

```diff
@@ -63,19 +63,25 @@ func TestV3NodeSearch(t *testing.T) {
 				t.Errorf("Could not run V3NodeSearch: %s", err)
 				continue
 			}
+
+			// Filter resp to top 20 matches to avoid flaky low matches.
+			topResp := &pbv3.NodeSearchResponse{
+				Nodes: resp.Nodes[:20],
```

Comment:
nit: Consider declaring the limit as a constant.

Reply:
Good idea, done